### PR TITLE
Feature: remove bundles

### PR DESCRIPTION
**Pull request by talyguryn**
Remove bundles for easy merging.

### CODE REVIEW

**Review by gohabereg — APPROVED** [review]

**Review by neSpecc — CHANGES_REQUESTED** [review]
Can not be merged without budling github actions

File: example/example-rtl.html [hunk 1]
```diff
@@ -27,6 +27,9 @@
   </div>
   <div class="ce-example__content _ce-example__content--small">
     <div id="editorjs"></div>
+    <div id="hint-core" style="text-align: center;">
+      No core bundle file found. Run <code class="inline-code">yarn build</code>
+    </div>
     <div id="hint" style="text-align: center;">
```
Comment (neSpecc) [inline example/example-rtl.html:33]:
```suggestion
    <div id="hint-tools" style="text-align: center;">
```

File: example/example-i18n.html [hunk 2]
```diff
@@ -58,7 +58,7 @@
 <script src="https://cdn.jsdelivr.net/npm/@editorjs/inline-code@latest"></script><!-- Inline Code -->
 
 <!-- Load Editor.js's Core -->
-<script src="../dist/editor.js"></script>
+<script src="../dist/editor.js" onload="document.getElementById('hint-core').hidden = true"></script>
```
Comment (neSpecc) [inline example/example-i18n.html:61]: where is the `hint-core` on this page?

**Review by neSpecc — CHANGES_REQUESTED** [review]

File: example/example-rtl.html (hunk 1 above)
Comment (talyguryn) [reply example/example-rtl.html:33]: thanks

File: example/example-i18n.html (hunk 2 above)
Comment (talyguryn) [reply example/example-i18n.html:61]: sorry. missed

**Review by neSpecc — APPROVED** [review]

**Review by neSpecc — APPROVED** [review]

**Pull request merged**